tests/test_h5store.py

```diff
@@ -50,15 +51,16 @@
 WINDOWS = (sys.platform == 'win32')
 
 
-@unittest.skipIf(not H5PY, 'test requires the h5py package')
-@unittest.skipIf(PYPY, 'h5py not reliable on PyPy platform')
-class BaseH5StoreTest(unittest.TestCase):
+@pytest.mark.skipif(not H5PY, reason='test requires the h5py package')
+@pytest.mark.skipif(PYPY, reason='h5py not reliable on PyPy platform')
+class TestBaseH5Store():
 
-    def setUp(self):
+    @pytest.fixture
+    def setUp(self,request):
```

Comment:
There is a space missing after the comma (also other occurrences).
```suggestion
    def setUp(self, request):
```
Please also fix this everywhere else where there is the same formatting issue.